### PR TITLE
Allow skipping keys that don't have message documentation (fixes #54)

### LINKED ISSUE #54: Allow to have a whitelist of keys that don't have message documentation
This would allow us to enable banana-checker in a lot more places, and ensure that any newly added messages have documentation (no regressing), while not failing entirely because some older messages are missing docs.

### PR DESCRIPTION
**Pull request by legoktm**
This would allow us to enable banana-checker on repositories that are
currently not fully documented, ensuring that we don't regress any
further when new messages are added, and can gradually come into proper
compliance.

A list of messages that should be skipped from checking can be specified
in the Gruntfile.js with the "skipIncompleteMessageDocumentation"
option.

Tests included.

### CODE REVIEW

File: test/skipIncompleteMessageDocumentation/qqq.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+	"@metadata": {
+		"first-metadata-key": "metadata value",
+		"second-metadata-key": "metadata value"
```
Comment (Krinkle) [inline test/skipIncompleteMessageDocumentation/qqq.json]: This second meta key doesn't seem relevant to the test.

Comment (legoktm) [inline test/skipIncompleteMessageDocumentation/qqq.json]: I just copied it from the "simple" directory

**Comment by Krinkle** [comment]
Verified manually in the other direction as well. Reminds me, we need #10.
